Formula/chntpw.rb

```diff
@@ -0,0 +1,21 @@
+class Chntpw < Formula
+  desc "Offline NT Password & Registry Editor"
+  homepage "https://pogostick.net/~pnh/ntpasswd/"
+  url "https://pogostick.net/~pnh/ntpasswd/chntpw-source-140201.zip"
+  sha256 "96e20905443e24cba2f21e51162df71dd993a1c02bfa12b1be2d0801a4ee2ccc"
+  license "GPL-2.0-only"
+
+  depends_on "openssl@1.1"
+
+  def install
+    ENV.append "CFLAGS", "-DUSEOPENSSL"
+    ENV.append "CFLAGS", "-I#{Formula["openssl@1.1"].opt_prefix}/include"
+
+    system "make", "chntpw", "cpnt", "reged", "samusrgrp", "sampasswd"
+    bin.install "chntpw", "cpnt", "reged", "samusrgrp", "sampasswd"
+  end
+
+  test do
+    assert_match "chntpw: change password of a user in a Windows SAM file", shell_output("#{bin}/chntpw -h")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula